docs/eep.schema

```diff
@@ -105,6 +105,10 @@
                                     "description": "Pull resistor setting for this gpio",
                                     "enum": [ "default", "up", "down", "none" ]
                                 }
+                                "comment": {
+                                    "description": "An optional comment describing the function of this gpio",
+                                    "type": "string",
```

Comment:
JSON doesn't allow a comma after the last key-value pair at the end of a block. Maybe the schema-validator works a little bit different, but please verify that.

https://www.json.org/json-en.html

Reply:
done